libs/parser/CMakeLists.txt

```diff
@@ -1 +1,2 @@
 add_library(parser STATIC parser.cpp)
+target_link_libraries(parser boost)
```

Comment:
We have a style guide to write it as
```CMake
target_link_libraries(parser 
    boost
    )
```
so that adding libraries does not affect the whole statement in diff. Please change the formatting.